# Specific Aims

It has been well established that the general reproducibility of biomedical
research is limited. There is a strong likelihood that it is particularly
problematic within the new area of microbiome research because there are a
growing number of scientists coming into the field who do not have adequate
training in performing informatics research. Furthermore, these self-trained
bioinformaticists are frequently isolated from other individuals who perform
similar analyses and they do not
have access to the growing number of resources needed to improve the
documentation and reproducibility of their analyses. Although they have been
well inculcated to perform detailed molecular biology experiments, many of those
cultural norms are not directly translatable to bioinformatics. Many of the
bioinformatics norms relate to the ability to make their analyses more
reproducible. One significant problem is that because the field is rapidly
growing and still quite new, no standard methods have been settled upon and it
is debatable whether there will ever be standardized methods. Therefore it is
difficult for researchers to know the correct way of analyzing their data and
to know the appropriate level of detail that should be included in their
manuscripts. A second problem is that bioinformatics research does not lend
itself well to using the traditional laboratory notebook, a critical
component of experimental reproducibility for a bench scientist. For the
microbiome researcher, the datasets are large, involve many steps at a computer,
and are iterative. Therefore it is cumbersome and perhaps even dangerous to
transcribe commands and results from a computer to a
paper-based notebook and back into the computer. These issues also limit the
value of most electronic laboratory notebooks. A third challenge is that
although it is common to share reagents such as bacterial strains or mice, the
concept of sharing data and analysis pipelines is novel to many of these
researchers. Yet their
datasets are large and can frequently be interrogated numerous ways beyond the
original design of the experiment. Thus, it is important that the data and analytical
methods be made publicly available so that others can extend the study beyond
the initial analysis. There is a clear need to develop instructional materials
for the microbiome research community to improve the reproducibility of its research.

The ***long-term goal*** of our research group is to understand the relationship
between the dynamics of the microbiome and human health. Towards that goal we
have a considerable track record in developing and disseminating the most widely
used software package being used in the area of microbiome research, mothur.
This has given us considerable insights into how microbiome researchers engage
in their research as we have interacted with them over the past decade. Based on
these observations, the ***objective*** of this proposal is to develop a
training module that members of the microbiome research community can use to
improve the reproducibility and overall quality of their data analysis. Given
our position within the microbiome research community, we are uniquely poised to
complete this objective and have unique insights into the best methods for
interacting with this community. Our ***rationale*** for why the proposed effort
is timely, is due to the significant growth of the community. To achieve our
objective, we propose the following specific aim:

**Develop a set of autotutorials to teach microbiome researchers habits for
engaging in reproducible research.** The goal of this module is to create
a framework within a research team for documenting research progress,
collaborating between researchers and supervisors, and ensuring the
reproducibility of their research as it is disseminated in publications and oral
presentations. These autotutorials will be developed, evaluated, and refined by
engaging microbiome researchers across a wide network that we have established
through our ongoing development of mothur and involvement in the microbiome
research community. The autotutorials will be the basis for establishing a
community-supported resource devoted to disseminating best practices in
microbiome analysis. The area of microbiome research is a relatively
young field that has been largely driven by advances in sequencing technology.
The past decade has witnessed significant changes in the field and it is equally
likely that even larger changes will occur over the next decade. Since traditional
curricula have not been able to keep up with these advances, there is an urgent
need to develop and disseminate such modules. Therefore, we will use this
project as a starting point to establish a resource that is supported by the
research community to develop and disseminate best practices as they relate to
performing microbiome research.

The ***expected outcomes*** of the proposed module are the improvement of the
reproducibility of research within the microbiome research community including
greater accessibility to original data, sequence analysis pipelines, and a
greater use of literate programming tools for constructing manuscripts and oral
presentations. Given the significant role of the microbiome in human health and
the advances in our understanding of how it shapes health and disease,
improving the reliability of the results form these studies will have a
meaningful ***positive impact***. Finally, we anticipate that the materials we
develop for the microbiome research community will be easily generalized to
other bioinformatics research disciplines.
